### PR TITLE
Add navbar dropdown changed

### PR DESCRIPTION
**Pull request by dovholuknf**
What do you think of this Ken
image

### CODE REVIEW

File: docusaurus/docusaurus.config.js [hunk 1]
```diff
@@ -505,33 +505,49 @@ const config = {
...
-                label: 'Twitter',
-                href: 'https://twitter.com/openziti',
+                type: 'html',
+                value: '<div class="text-divider"><p>Other</p></div>'
               },
               {
-                label: 'Ziggy\'s Twitter',
-                href: 'https://twitter.com/openziggy',
+                type: 'html',
+                value: '<span id="navbarDropdownItem"><img id="navbarDropdownImage" src="img/oz-test-kitchen.png"/><a href="https://github.com/openziti-test-kitchen" target="_blank">Open Ziti Test Kitchen</span></a>'
```
Comment (qrkourier) [inline docusaurus/docusaurus.config.js:537]: Consistency: "Open Ziti" vs "OpenZiti"

File: docusaurus/docusaurus.config.js [hunk 2]
```diff
@@ -505,33 +505,49 @@ const config = {
...
+                value: '<a href="https://zeds.openziti.org" target="_blank"><span id="navbarDropdownItem"><img id="navbarDropdownImage" src="img/zeds.png"/>ZEDS</span></a>'
+              },
+              {
+                type: 'html',
+                value: '<a href="https://netfoundry.io/pricing/" target="_blank"><span id="navbarDropdownItem"><img id="navbarDropdownImage" src="img/nf.svg"/>CloudZiti</span></a>'
+              },
+              {
+                type: 'html',
+                value: '<a href="https://www.linkedin.com/company/netfoundry/" target="_blank"><span id="navbarDropdownItem"><img id="navbarDropdownImage" src="img/li.svg"/>NetFoundry</span></a>'
+              },
```
Comment (qrkourier) [inline docusaurus/docusaurus.config.js:550]: Those two existing socials (Discourse, GitHub) are probably still the two most important, but part of me wants to present all of them the same way. Here's how it looks on mobile. 

Screenshot_20221209-192917

**Review by qrkourier — APPROVED** [review]
Nice!

File: docusaurus/docusaurus.config.js (hunk 1 above)
Comment (dovholuknf) [reply docusaurus/docusaurus.config.js:537]: egad how'd a space get in there! :) thx

**Pull request closed without merging**